packages/gestalt/src/Masonry.tsx

```diff
@@ -608,11 +637,12 @@ export default class Masonry<T> extends ReactComponent<Props<T>, State<T>> {
       const itemsToMeasureCount = hasMultiColumnItems
         ? MULTI_COL_ITEMS_MEASURE_BATCH_SIZE + 1
         : minCols;
-      const itemsToMeasure = items
+
+      const itemsToMeasure = (items.length === 0 ? _loadingStateItems : items)
```

Comment:
we should never be measuring loadingStateItems imo since we have the ability to assign the height ahead of time